bin/strap.sh

```diff
@@ -413,69 +452,56 @@ if [ ! -d "$STRAP_SRC_DIR" ]; then
   git clone $Q "git@github.com:daptiv/strap" "$STRAP_SRC_DIR"
 else
   (
-    logdebug "Updating local repository."
+    log "Updating local repository."
     cd "$STRAP_SRC_DIR"
     git pull $Q --rebase --autostash
   )
 fi
 logk
 
-# Setup dotfiles
```

Comment:
Is this done somewhere else now?

Reply:
the section does both, with the user dot files having a fallback to daptiv in case the active user doesn't have their own.  addressed in PR 40